cl/donate/urls.py

```diff
@@ -1,61 +1,17 @@
 from django.urls import path
+from django.views.generic.base import RedirectView
 
-from cl.donate.paypal import donate_paypal_cancel, process_paypal_callback
 from cl.donate.stripe_helpers import process_stripe_callback
-from cl.donate.views import (
-    badge_signup,
-    cc_payment,
-    donate,
-    make_check_donation,
-    payment_complete,
-    toggle_monthly_donation,
-)
-from cl.users.views import view_donations
 
 urlpatterns = [
-    # Donations & payments
-    path("donate/", donate, name="donate"),
-    path("pay/", cc_payment, name="cc_payment"),
-    path("badges/sign-up/", badge_signup, name="badge_signup"),
     path(
-        "donate/complete/",
-        payment_complete,
-        {"template_name": "donate_complete.html"},
-        name="donate_complete",
-    ),
-    path(
-        "pay/complete/",
-        payment_complete,
-        {"template_name": "payment_complete.html"},
-        name="payment_complete",
-    ),
-    path(
-        "badges/complete/",
-        payment_complete,
-        {"template_name": "badge_signup_complete.html"},
-        name="badge_signup_complete",
-    ),
-    # Paypal
-    path("donate/paypal/cancel/", donate_paypal_cancel, name="paypal_cancel"),
-    path(
-        "donate/callbacks/paypal/",
-        process_paypal_callback,
-        name="paypal_callback",
+        "donate/",
+        RedirectView.as_view(url="https://free.law/donate"),
```

Comment:
I usually put all our one-off legacy redirects in cl/urls.py. If we do that here, we can remove this file once the stripe callback is removed.

Reply:
> I usually put all our one-off legacy redirects in cl/urls.py.

understood!